scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -1,6 +1,6 @@
 {% macro igv_track_selection(igv_tracks, current_user) %}
 <div id="accordion">
-  <a data-toggle="collapse" data-parent="#accordion" href="#track_settings">Settings</a>
+  <a data-toggle="collapse" data-parent="#accordion" href="#track_settings"><i class="fas fa-cogs text-secondary"></i></a>
```

Comment:
Nice! As it is a little to the side, of the button, and drops down to get close to the RNA button, probably keep a label anyway, but make it like "Settings DNA" or something that makes it distinct from the junctions track?